src/main/java/organice/model/person/UniquePersonList.java

```diff
@@ -36,6 +36,15 @@ public boolean contains(Person toCheck) {
         return internalList.stream().anyMatch(toCheck::isSamePerson);
     }
 
+    /**
+     * Returns true if the list contains an equivalent doctor in charge as the given argument.
+     */
+    public boolean containsDoctor(Nric doctor) {
+        requireNonNull(doctor);
+        return internalList.stream().anyMatch(n -> n.getType().toString().equals("doctor")
```

Comment:
There is no need to convert the `Type` object to a `String` before comparing. There is a `isDoctor` method in Type.java which can be used to determine if it is a `Doctor`.